exec.py

```diff
@@ -124,47 +145,139 @@ def plot_plane(ws, tag):
 
     fig.savefig(f"output/test_{tag}.png", dpi=300)
 
+def divide_into_chunks(args):
+    '''Divide the number of events in the input file into given chunk sizes.'''
+    # Read the input file
+    infile = uproot.open(args.inpath)
```

Comment:
seems like there's something missing here

Reply:
What do you think is missing here? Chunksize is also embedded into the args, if you were thinking about that. Or is there anything else you were thinking about, thank you!

Reply:
Sorry, I got confused by the way github shows this part of the diff. When I looked at it before, it seemed to me like the function ended after the `infile` line, i.e. before doing anything. In reality, it's fine.